src/YouTrackSharp/TimeTracking/TimeSettings.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace YouTrackSharp.TimeTracking
+{
+	/// <summary>
+	/// A class that represents YouTrack time settings information.
+	/// </summary>
+	public class TimeSettings
+	{
+		/// <summary>
+		/// Creates an instance of the <see cref="TimeSettings" /> class.
+		/// </summary>
+		public TimeSettings()
+		{
+		}
+
+		/// <summary>
+		/// Hours A Day
+		/// </summary>
+		[JsonProperty("hoursADay")]
+		public int HoursADay { get; set; }
+
+		/// <summary>
+		/// Days A Week
+		/// </summary>
+		[JsonProperty("daysAWeek")]
+		public int DaysAWeek { get; set; }
+
+		[JsonProperty("workWeek")]
+		public List<WorkDay> Workweek { get; set; }
+
+		public class WorkDay
```

Comment:
Chack if this class can be removed and `SubValue<int>` can be used instead?
(https://github.com/JetBrains/YouTrackSharp/blob/master/src/YouTrackSharp/Entities/SubValue.cs)